server/common/thrift/src/main/thrift-java/org/kaaproject/kaa/server/common/thrift/gen/operations/RouteInfo.java

```diff
@@ -503,7 +503,7 @@ public String toString() {
     if (this.endpointId == null) {
       sb.append("null");
     } else {
-      sb.append(this.endpointId);
+      sb.append(Hex.encodeHexString(this.endpointId.array()));
```

Comment:
This is autogenerated class your changes will be overwritten after next generation.